src/plugins/intel_gpu/src/graph/graph_optimizer/remove_redundant_reorders.cpp

```diff
@@ -235,7 +235,8 @@ void remove_redundant_reorders::run(program& p) {
             continue;
         }
 
-        if (!o_layout.compatible(i_layout))
+        if (!o_layout.compatible(i_layout) ||
```

Comment:
We need to check global impact of this change. Please trigger benchmark perf check

Reply:
I delete this code and find other way to solve issue.